nailgun/entities.py

```diff
@@ -1853,15 +1853,23 @@ class HostCollection(
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
             'description': entity_fields.StringField(),
-            'max_content_hosts': entity_fields.IntegerField(),
+            'max_hosts': entity_fields.IntegerField(),
             'name': entity_fields.StringField(required=True),
             'organization': entity_fields.OneToOneField(
                 Organization,
                 required=True,
             ),
             'system': entity_fields.OneToManyField(System),
-            'unlimited_content_hosts': entity_fields.BooleanField(),
+            'unlimited_hosts': entity_fields.BooleanField(),
         }
+        # The following attributes have been renamed with Satellite
+        # 6.2, so we revert them to their old values if we have an
+        # older version of Satellite
+        if _get_version(server_config) <= Version('6.1'):
```

Comment:
```python
>>> from packaging.version import parse
>>> parse('6.1.7') <= parse('6.1')
False
>>> parse('6.1.7') < parse('6.2')
True
```
`6.1.7` is obviously bigger than `6.1` so the check won't work, please, change this line to
```python
        if _get_version(server_config) < Version('6.2'):
```